documentation/docs/30-advanced/25-errors.md

```diff
@@ -94,7 +94,7 @@ export function handleError({ error, event }) {
 
 	return {
 		message: 'Whoops!',
-		code: error.code ?? 'UNKNOWN'
+		code: error?.code ?? 'UNKNOWN'
```

Comment:
I just created a new project, upgraded to TypeScript 4.9, and created a hook with this code. It's giving an error on this line telling me `error` is `unknown`

Reply:
It's stupid, but the TypeScript folks intend for this not to work: https://github.com/microsoft/TypeScript/issues/35799

Reply:
I can only get this to work is either changing the type to `any`:
```
/** @type {import('@sveltejs/kit').HandleServerError} */
export function handleError({ error, event }) {
	/** @type {any} */
	const err = error;
	return {
		message: 'Whoops!',
		code: err?.code ?? 'UNKNOWN'
	};
}
```

Or checking if the type is an `Error` and saying it might have a `code`:
```
/** @type {import('@sveltejs/kit').HandleServerError} */
export function handleError({ error, event }) {
	if (!(error instanceof Error)) {
		throw new Error('Invalid error type ' + error);
	}
	/** @type {Error & {code?: string}} */
	const err = error;
	return {
		message: 'Whoops!',
		code: err?.code ?? 'UNKNOWN'
	};
}
```